src/components/visualizations/chart/highcharts/test/comboConfiguration.spec.ts

```diff
@@ -36,4 +37,26 @@ describe("Combo Configuration", () => {
             expect(getDefaultChartType(config)).toEqual(LINE);
         });
     });
+
+    it("should disable saturate other for combo line_line", () => {
+        const config = {
+            primaryChartType: LINE,
+            secondaryChartType: LINE,
+        };
+        const {
+            plotOptions: { series: series },
+        } = getComboConfiguration(config);
+        expect(series.states.inactive.opacity).toBe(1);
+    });
+
+    it("should not disable saturate other elements for another kind of combo", () => {
+        const config = {
+            primaryChartType: COLUMN,
+            secondaryChartType: LINE,
+        };
+        const {
+            plotOptions: { series: series },
+        } = getComboConfiguration(config);
+        expect(get(series, "states.inactive.opacity")).not.toBe(1);
+    });
```

Comment:
how about adding more test with empty `config`, cause it is optional

Reply:
done